TPP.Persistence.MongoDB/Repos/BadgeRepo.cs

```diff
@@ -64,26 +68,55 @@ private void InitIndexes()
         }
```

Comment:
(above here, can't comment on the correct line).
Please add indexes for the new filterable fields. 
Nitpick: Also this is fine for now, but I suspect filtering for "!= true" for the shiny field _might_ not work properly for the index. Maybe it does, but it's something you can test if you're still motivated. E.g. by enabling [profiling level 2](https://docs.mongodb.com/manual/tutorial/manage-the-database-profiler/), issuing a query, and looking at the execution plan.